src/main/webapp/js/feed-loader.js

```diff
@@ -0,0 +1,68 @@
+// Fetch messages and add them to the page.
+function fetchMessages(){
+const url = '/feed';
+fetch(url).then((response) => {
+  return response.json();
+}).then((messages) => {
+  const messageContainer = document.getElementById('message-container');
+  if(messages.length == 0){
+   messageContainer.innerHTML = '<p>There are no posts yet.</p>';
+  }
+  else{
+   messageContainer.innerHTML = '';
+  }
+  messages.forEach((message) => {
+   const messageDiv = buildMessageDiv(message);
+   messageContainer.appendChild(messageDiv);
+  });
+});
+}
+
+function buildMessageDiv(message){
+const usernameDiv = document.createElement('div');
```

Comment:
here too, indenting looks off.

Reply:
basically, indent everything in a method so that it is easy to see where the method ends.